add-ons/dspace-docker-tools/ingestAIPd4.sh

```diff
@@ -3,10 +3,14 @@
 CHECKFILE=/dspace/assetstore/ingest.hasrun.flag
 
 export JAVA_OPTS="${JAVA_OPTS} -Xmx2500m -Dupload.temp.dir=/dspace/upload -Djava.io.tmpdir=/tmp"
+echo ${LOCALCFG1} >> /dspace/config/local.cfg
```

Comment:
We probably should add a comment like:
\# that is currently the best way to extend the configuration. It might result having these lines added multiple times to local.cfg. Once we figured out how ro use environment variables to change our configuration we should change this

Reply:
@terrywbrady and @pnbecker : I actually don't understand what is going on here.  Why are we appending three *different* variables (`LOCALCFG1`, `LOCALCFG2`, `LOCALCFG3`) onto the end of the `local.cfg`?  Why split this in three variables?  It seems to almost imply (to me) that these variables are standalone, but (as we know) if you accidentally add the same configuration property to both `LOCALCFG1` and `LOCALCFG2`, then you'll end up with an array of values for that configuration property.

I'm not sure what advice to add to this ticket, as I don't quite understand the purpose of these variables -- in fact `LOCALCFG1` and `LOCALCFG2` don't seem to ever be set (to anything useful) in this PR.  Only `LOCALCFG3` is used for RDF related settings (which could instead be passed in as System properties via JAVA_OPTS, similar to Oracle settings)

Reply:
Resolving.  Switched to system properties instead.